client/lib/analytics/statsd-utils.ts

```diff
@@ -49,13 +49,13 @@ export function createStatsdURL( calypsoSection: string, events: BeaconData[] |
 /**
  * Logs server events to statsd. Uses superagent for server-side network requests
  * and can disable statsd events with the server_side_boom_analytics_enabled
- * config flag.
+ * config flag. Does nothing if called on the client.
  *
  * @param calypsoSection The Calypso section the event occurred under.
  * @param events List of events to send to the server.
  */
 export function logServerEvent( calypsoSection: string, events: BeaconData[] | BeaconData ) {
-	if ( config( 'server_side_boom_analytics_enabled' ) ) {
+	if ( typeof window === 'undefined' && config( 'server_side_boom_analytics_enabled' ) ) {
```

Comment:
Some of the theme fetch functions can also be called on the client. For now, we just want to see server stats here. (Noting that we currently have no statsd events on the client anyways.)